### ISSUE TITLE
How to get started?

### ISSUE THREAD

**Issue by SJTUGuofei** [Santara/anirban-imitation#1]
Dear Anirban:

For this "imitation" project, I'm confused about how to get started and there are also no any introduction documentations exist.
Could you please explain how the whole project worked?
What's the detailed dependencies for running the project? For now, I run the project on Ubuntu 14.04, with CUDA, cudnn, Tensorflow, Gym installed. 

Thank you!
Guofei

**Comment by Santara** [comment]
Hi, This is a fork of the repository: https://github.com/openai/imitation 
Also currently it uses Theano for neural networks, not Tensorflow.
You can check the file COMMANDS for getting started.

**Comment by SJTUGuofei** [comment]
Thank you so much for your kindly response.

When I running the code following the COMMANDS, there are some error come out:

IOError: Unable to open file (Unable to open file: name = 'trajectories/expert_trajectories-cartpole-v0', errno = 2, error message = 'no such file or directory', flags = 0, o_flags = 0)

In this or in the original "imitation" repository, there are not any directory called "trajectories", so I'm confused that is there missing some files in this repository?

**Comment by Santara** [comment]
yes, you need to create the necessary extra directories. I added directories with heavy files in .gitignore. I recommend you to run code from the original repository from OpenAI: https://github.com/openai/imitation
